src/components/accordion/_accordion.scss

```diff
@@ -21,6 +25,9 @@
     border: none;
     border-bottom: $border-light;
     border-width: 2px;
+    justify-content: space-between;
+    margin-right: unset;
```

Comment:
replace with inline-inset-start

Reply:
Done. Have shifted this code to description mixin. As it should apply wherever info icon is there, and some elements which don't use input mixin were left.